src/modules/types/compositeAddress.ts

```diff
@@ -1,4 +1,4 @@
 export interface ICompositeAddress {
-    address?: string;
+    address: string;
```

Comment:
Could you update the `ProposalDataListItemStructure` component to reflect these changes?